File: time_logic.cpp
#include <iostream>
#include <String>
using namespace std;
int main() {
    int y;
    cout << "Enter an amount of seconds to convert:" << endl;
    cin >> y;
    int seconds = y%60;
    int totalMinutes = y/60;
    int minutes = totalMinutes%60;
    int hours = totalMinutes/60;
    cout << y << " seconds is " << hours << " hours, " << minutes << " minutes, and " << seconds << " seconds." << endl;
    return 0;
}

File: Lab1/time_logic.py
x = input("Enter an amount of seconds to convert:")
y = int(x)
seconds = y%60
totalMinutes = y//60
minutes = totalMinutes%60
hours = totalMinutes//60
print(y , "seconds is " , hours , " hours, " , minutes , " minutes, and " , seconds , " seconds.")
